1-exercises/B-todo-list/script.js

```diff
@@ -1,12 +1,32 @@
 function todoList(todos) {
   // Write your code here...
+  const mainDiv = document.querySelector("#content");
+  const todoList = document.createElement("ul");
+
+  let toDoItems = todos
+    .map((todo) => {
+      return `
+    <li>${todo.todo}</li>
+    `;
+    })
+    .join("");
+
+  todoList.innerHTML = toDoItems;
+  mainDiv.append(todoList);
+
+  let items = document.querySelectorAll("li");
+  items.forEach((item) => {
+    item.addEventListener("click", () => {
+      item.classList.toggle("strike");
```

Comment:
Looks great! We could also add a css class to each element and change the hover to have a `cursor:pointer` for a slightly better UI!